### PR TITLE
Add tests for Glucose

### PR DESCRIPTION
**Pull request by EmmanuelAB**

### CODE REVIEW

File: app/src/androidTest/java/com/project/symptoms/db/dao/GlucoseDaoImplTest.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
+package com.project.symptoms.db.dao;
+
```
Comment (xsaco07) [inline app/src/androidTest/java/com/project/symptoms/db/dao/GlucoseDaoImplTest.java:2]: Why the test class is not in test directory?

File: app/src/main/java/com/project/symptoms/db/dao/GlucoseDaoImpl.java [hunk 2]
```diff
@@ -32,7 +32,6 @@ public long insert(GlucoseModel glucoseModel) throws Exception {
 
         // Insert the new row, returning the primary key value of the new row
         long newRowId = db.insert(Contract.Glucose.TABLE_NAME, null, values);
-        db.close();
```
Comment (xsaco07) [inline app/src/main/java/com/project/symptoms/db/dao/GlucoseDaoImpl.java:35]: Why this line was deleted? Shouldn't we close the DB anytime is opened to perform any operation?

File: app/src/test/java/com/project/symptoms/db/controller/GlucoseControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
+package com.project.symptoms.db.controller;
+
```
Comment (xsaco07) [inline app/src/test/java/com/project/symptoms/db/controller/GlucoseControllerTest.java:2]: Why the test class is not in test directory?

File: app/src/test/java/com/project/symptoms/db/controller/GlucoseControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    @Before
+    public void setup(){
+        mockDao = Mockito.mock(GlucoseDaoImpl.class);
+        controller = GlucoseController.getInstance(null);
+        controller.setGlucoseDao(mockDao);
+    }
+
+    @Test
+    public void GivenSomeRecordsWhenSelectAllReturnsAList() throws Exception{
```
Comment (xsaco07) [inline app/src/test/java/com/project/symptoms/db/controller/GlucoseControllerTest.java]: Consider to add the word "test" somewhere in the name to follow the convention. 
The same in the other tests.

File: app/src/androidTest/java/com/project/symptoms/db/dao/GlucoseDaoImplTest.java (hunk 1 above)
Comment (EmmanuelAB) [reply app/src/androidTest/java/com/project/symptoms/db/dao/GlucoseDaoImplTest.java:2]: because this test runs using the real app context

**Review by xsaco07 — APPROVED** [review]

**Pull request merged**